easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb

```diff
@@ -0,0 +1,17 @@
+name = 'COMSOL'
+version = '5.2a'
+
+homepage = 'https://www.comsol.se/'
+description = """COMSOL Multiphysics - Simulation Tool for Electrical, Mechanical, Fluid Flow, and Chemical
+Applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
I think it's worth including a comment on how to obtain this tarball?

I think COMSOL typically comes as a `.iso`? Is it clear which tarball to grab from that, or should we specify it more clearly?

Also, please include a SHA256 checksum?